client/signup/steps/page-picker/index.tsx

```diff
@@ -0,0 +1,233 @@
+import { Button } from '@automattic/components';
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import { BrowserView } from 'calypso/signup/difm/components/BrowserView';
+import {
+	HOME_PAGE,
+	BLOG_PAGE,
+	CONTACT_PAGE,
+	ABOUT_PAGE,
+	PHOTO_GALLERY_PAGE,
+	SERVICE_SHOWCASE_PAGE,
+	VIDEO_GALLERY_PAGE,
+	PODCAST_PAGE,
+	PORTFOLIO_PAGE,
+	FAQ_PAGE,
+	SITEMAP_PAGE,
+	PROFILE_PAGE,
+	PAGE_LIMIT,
+} from 'calypso/signup/difm/constants';
+import { useTranslatedPageTitles } from 'calypso/signup/difm/translation-hooks';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import './style.scss';
+
+const PageGrid = styled.div`
+	display: grid;
+	grid-template-columns: 1fr;
+	row-gap: 20px;
+	column-gap: 35px;
+	margin: 0 0 30px;
+
+	@media ( min-width: 960px ) and ( max-width: 1200px ) {
+		grid-template-columns: 1fr 1fr;
+		column-gap: 15px;
+		row-gap: 25px;
+	}
+
+	@media ( min-width: 1200px ) {
+		grid-template-columns: 1fr 1fr 1fr;
+		row-gap: 40px;
+		column-gap: 35px;
+	}
+`;
+
+const GridCellContainer = styled.div< { isClickDisabled: boolean; isSelected: boolean } >`
+	cursor: ${ ( { isClickDisabled, isSelected } ) =>
+		! isSelected && isClickDisabled ? 'default' : 'pointer' };
+	opacity: ${ ( { isSelected, isClickDisabled } ) =>
+		! isSelected && isClickDisabled ? '0.4' : '1' };
+	border-radius: 4px;
+	position: relative;
+	width: 100%;
+	position: relative;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	font-weight: 500;
+`;
+
+const CellLabelContainer = styled.div`
+	margin: 14px 0;
+	text-align: left;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	font-size: 14px;
+	margin-right: 8px;
```

Comment:
```suggestion
	gap: 8px;
```
I think you meant to use `gap` here. (Image below)
<img width="183" alt="image" src="https://user-images.githubusercontent.com/5436027/159427173-932cce0a-a6ef-4ac0-9eac-db7a9ba9d950.png">

Reply:
Ah TIL!